lib/domain/upvote.rb

```diff
@@ -0,0 +1,23 @@
+module Domain
+  class Upvote
```

Comment:
Why Upvote as aggregate? Is it really an aggregate here? What problem shall this PR solve?